addons/website/static/src/js/editor/wysiwyg_multizone.js

```diff
@@ -256,6 +256,21 @@ var WysiwygMultizone = Wysiwyg.extend({
             }.bind(this));
         }.bind(this));
     },
+    /**
+     * Setup editable attributes on the given element.
+     * This is used when element is updated/replaced
+     * after the editor is initialized.
+     *
+     * @public
+     * @param {jQuery} $el
+     */
+    setupEditable: function ($target) {
+        $target.find('.note-editable').addClass('o_not_editable').attr('contenteditable', false);
+        $target.find(this.selectorEditableArea, this._summernote.layoutInfo.editable).attr('contenteditable', true);
+        $target.find('[data-oe-readonly]').addClass('o_not_editable').attr('contenteditable', false);
+        $target.find('.oe_structure').attr('contenteditable', false).addClass('o_fake_not_editable');
+        $target.find('[data-oe-field][data-oe-type="image"]').attr('contenteditable', false).addClass('o_fake_not_editable');
```

Comment:
Isn't there already a method of the editor which does that ? How are these classes and attributes appearing in the normal case ?

Reply:
I don't think there is such method, The attributes ares setup only once when editor is initialized.
https://github.com/odoo/odoo/blob/master/addons/website/static/src/js/editor/wysiwyg_multizone.js#L198-L202
If you want, i can split the code in separeate method instead of rewriting.

Reply:
Yes, it will probably be better, thanks ;)